Bottom: 15e172508537211de42e954148ccfcbf4eb510be
Top:    178c82f1375df67a383ae610def35a42dae37d43
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 01:41:21 +0000

Refresh of board-outline-polygon-generati

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index bc9eb66..b604e05 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1371,9 +1371,8 @@ GhidDrawMask (int side, BoxType * screen)
   gui->graphics->set_color (out->fgGC, PCB->MaskColor);
   ghid_set_alpha_mult (out->fgGC, thin ? 0.35 : 1.0);
 
+  memset (&polygon, 0, sizeof (polygon));
   polygon.Clipped = board_outline_poly ();
-  polygon.NoHoles = NULL;
-  polygon.NoHolesValid = 0;
   polygon.BoundingBox = *screen;
   polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
